presto-product-tests/conf/docker/acid/compose.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
When is this environment run?

Anyway, https://github.com/prestosql/presto/pull/1239 will make test scaffolding sane here.

Reply:
This had to added to scripts, I was running it by hand but I see with your change it will be default. Will wait for #1239 to merge and then remove this env.